src/integration/pages/articlePage/pidgin.test.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * @pathname /pidgin/tori-51745682
+ */
+
+import runCanonicalUserTests from './user.canonical';
+import runAmpUserTests from './user.amp';
+import {
+  runFooterTests,
+  runHeaderTests,
+  runCommonSeoTests,
+  runCommonCanonicalAnalyticsTests,
+  runCommonAmpAnalyticsTests,
+  runCommonA11yTests,
+  runCoreAmpTests,
+  runCoreCanonicalTests,
+  runPerformaceTests,
+  runSnapshotTests,
+} from '../../common';
+
+describe('Given I am on a Pidgin article AMP page', () => {
```

Comment:
In future, it would be good to not have both a pidgin and mundo article with integration tests, but just one left-to-right script service, one right-to-left script service and one with a variant. e.g. `mundo` `arabic` `ukchina`

Reply:
I think these were just included as part of the experiment to ensure that all page types would work OK. The expectation is, that once these tests are running reliably (and quickly!) in the CI pipeline - the Media team were happy to trial it out for our page types (live radio & MAPs) first, the other teams would then implement the tests for their own respective page types.  At that point, you could delete the pidgin test & replace with the services suggested above.

Reply:
Thanks for explaining. That sounds like a good plan.